src/message/MessageContainer.js

```diff
@@ -64,7 +64,7 @@ export default class MessageContainer extends React.PureComponent {
       pushRoute,
     } = this.props;
     const MessageComponent = isBrief ? MessageBrief : MessageFull;
-    const childrenNodes = htmlToDomTree(message.match_content || message.content);
+    const childrenNodes = htmlToDomTree((message.match_content || message.content).replace(/\r\n/g, '<br />').replace(/[\r\n]/g, '<br />'));
```

Comment:
I am applying `flexDirection: 'row',` to `pre` tag. Due to this, all content is aligned to a single line (rendered in same line with scrolling). And by using this detecting when to break line (code indentation, new lines etc)